test/factories/host_related.rb

```diff
@@ -65,7 +65,7 @@ def set_nic_attributes(host, attributes, evaluator)
 
   factory :nic_managed, :class => Nic::Managed, :parent => :nic_interface do
     type 'Nic::Managed'
-    sequence(:mac) { |n| "01:23:45:ab:" + n.to_s(16).rjust(4, '0').insert(2, ':') }
+    sequence(:mac) { |n| "00:23:45:ab:" + n.to_s(16).rjust(4, '0').insert(2, ':') }
```

Comment:
Can you change one of the other bytes to ensure this sequence doesn't overlap with Nic::Base's above? I think that's why the first part was changed to 01. Note that the other factory sequences use 23:56, 23:67 etc to avoid duplicating.